src/crimson/os/seastore/cache.cc

```diff
@@ -245,6 +246,77 @@ void Cache::register_metrics()
       ),
     }
   );
+
+  /*
+   * cache_query: cache_access and cache_hit
+   */
```

Comment:
nit, 
```c++
// cache_query: cache_access and cache_hit
```

Reply:
I wanted to use /* ... */ to emphasize the section of metrics to be registered.